app/forms/upload_form.py

```diff
@@ -0,0 +1,14 @@
+from flask_wtf import FlaskForm
+from wtforms import SubmitField, SelectField
+from wtforms.validators import DataRequired
+
+from config import Configuration
+
+conf = Configuration()
+
+
+class UploadForm(FlaskForm):
+    model = SelectField('model', choices=conf.models, validators=[DataRequired()])
+    warning = False
```

Comment:
what are these lines for? Maybe you can add a comment in the code to clarify why they are needed, otherwise developers might remove them in the future as they seem unnecessary